packages/react/src/auto/AutoTableContext.ts

```diff
@@ -0,0 +1,30 @@
+import { GadgetRecord } from "@gadgetinc/api-client-core";
+import React, { useContext } from "react";
+import type { TableResult } from "src/useTableUtils/types.js";
+
+export type AutoTableContext = TableResult<GadgetRecord<any>>;
+
+/**
+ * React context that stores an instance of the metadata loaded for a particular form action
+ */
+
+export const AutoTableContext = React.createContext<AutoTableContext | undefined>(undefined);
+/**
+ * Get the current `metadata` object and `submit` function from React context
+ * Must be called within a component wrapped by the `<AutoForm>` component.
+ **/
+
+export const useAutoTableContext = () => {
```

Comment:
Super small nit: it looks nicer not to have an empty line between the comment block and the variable.